### PR TITLE
Tdd

### PR DESCRIPTION
**Pull request by richardoppiyo**
In this project we used TDD to build the following class and methods:

- Solver
  - factorial
  - reverse
  - fizzbuzz

### CODE REVIEW

File: spec/reverse_spec.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require './solver'
+
+describe 'the reverse of a string' do
+  before :each do
+    @solver = Solver.new
+  end
+
+  it 'should return the reverse of the given string' do
+    # fact = Solver.new.factorial(3)
```
Comment (rustam-tolipov) [inline spec/reverse_spec.rb]: - [x] Please delete commits that are not necessary to use.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,58 @@
...
+
+ Give a ⭐️ if you like this project!
+
+ ## Acknowledgments
+
+ - Microverse: [microverse community](https://github.com/microverseinc)
+
+ ## 📝 License
+
+ This project is [MIT](./MIT.md) licensed.
```
Comment (rustam-tolipov) [inline README.md:58]: - [x] I can see you added LICENSE to your project. However when I try to open it by Readme.md link it's not working as expected. Kindly connect your LICENSE file to your Readme.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @richardoppiyo, 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Kakalanp) [reply README.md:58]: It is because of the license's name :sweat_smile:
Fixing it right now

File: spec/reverse_spec.rb (hunk 1 above)
Comment (Kakalanp) [reply spec/reverse_spec.rb]: We didn't notice at first, thanks! :100:

**Review by rotimiazeez — APPROVED** [review]
# Hi @richardoppiyo 💪🏽, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kakalanp** [comment]
Thank you! Merging right away! :tada:

**Pull request merged**